charts/values.yaml

```diff
@@ -64,3 +79,277 @@ engines:
   replicated:
     mayastor:
       enabled: true
+
+loki:
+  enabled: true
+  # Configuration for loki's localpv hostpath storage class.
+  localpvScConfig:
+    enabled: true
+    loki:
+      # Name of loki's localpv hostpath storage class.
+      name: "openebs-loki-localpv"
+      # -- Host path where local loki data is stored in.
+      basePath: "/var/local/{{ .Release.Name }}/localpv-hostpath/loki"
+      # -- ReclaimPolicy of loki's localpv hostpath storage class.
+      reclaimPolicy: Delete
+      # -- VolumeBindingMode of loki's localpv hostpath storage class.
+      volumeBindingMode: WaitForFirstConsumer
+    minio:
+      # Name of minio's localpv hostpath storage class.
+      name: "openebs-minio-localpv"
+      # -- Host path where local minio data is stored in.
+      basePath: "/var/local/{{ .Release.Name }}/localpv-hostpath/minio"
+      # -- ReclaimPolicy of minio's localpv hostpath storage class.
+      reclaimPolicy: Delete
+      # -- VolumeBindingMode of minio's localpv hostpath storage class.
+      volumeBindingMode: WaitForFirstConsumer
+
+  loki:
+    schemaConfig:
+      configs:
+        - from: 2024-04-01
+          store: tsdb
+          object_store: s3
+          schema: v13
+          index:
+            prefix: loki_index_
+            period: 24h
+    commonConfig:
+      replication_factor: 3
+
+    # Configure these if you don't want to use minio and rather an external s3 bucket.
```

Comment:
Can we point to documentation url instead?

Reply:
I initially thought would do that, but the documentation is not very clear and the options are segregated and not in one place.

Reply:
Maybe we can put this in a doc and then just point to it

Reply:
Sure. I will add a document for the storage configurations and point it to this. Will take it up in the next PR.

Reply:
Done.